src/main/ComputerDecision.java

```diff
@@ -0,0 +1,110 @@
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Iterator;
+
+public class ComputerDecision {
+    private Card[] hand;
+    private Card[] flop;
+    private int callAmount;
+
+    public ComputerPlayerDecisionClass(Card[] hand, Card[] flop, int callAmount) {
```

Comment:
Would probably want to move this to an entity interface and factory system if the decision is created on each new round.

Reply:
I think making the class static could work, we don't really need different instances using the same algorithm, so I just need to make a static method that takes parameters for the cards.